internal/perms/acl_test.go

```diff
@@ -777,6 +779,31 @@ func TestACL_ListPermissions(t *testing.T) {
 				},
 			},
 		},
+		{
+			name:         "separate_type_id_resource_grants",
+			scopes:       map[string]*scopes.ScopeInfo{"p_1": nil},
+			resourceType: resource.Target,
+			actionSet:    action.ActionSet{action.Read, action.Cancel},
+			aclGrants: []scopeGrant{
+				{
+					scope: "p_1",
+					grants: []string{
+						"type=target;actions=list",
+						"id=ttcp_1234567890;actions=read",
```

Comment:
I see the action part of this grant didn't make it into the permissions, is this an invalid grant? Should there not be a "read" action too?